### DISCUSSION TITLE
where is that distilled model? why does it say gpt-3.5-turbo:finetuned-123 in the sample code?

### DISCUSSION THREAD

**Discussion by bhomass** [567-labs/instructor#811] [Q&A] [answered]
I run instructor jobs create-from-file math_finetunes.jsonl many times, I don't see any fine tuned model created.

what about in the chain of density demo. there were many fine tuned models, I don't see any instructions for naming the different distill models.

**Comment by bhomass** [comment, marked as answer]
found it in openai platform.

**Comment by jxnl** [comment]
@ivanleomk can we update documentation?
